git_stacktrace/git.py

```diff
@@ -149,8 +150,8 @@ def line_match(commit, traceback_line):
 
 def format_one_commit(commit):
     result = []
-    custom, full, url = get_commit_info(commit)
-    result.append(custom)
+    summary, _, _, url, _, _ = get_commit_info(commit)
```

Comment:
This could get a little nuts so maybe adopt this notation:

```python
summary, _a, _b, url, _c, _d = get_commit_info(commit)
```

... replacing `_a`, etc. with more specific names.